src/qtism/data/rules/BranchRule.php

```diff
@@ -80,17 +80,26 @@ class BranchRule extends QtiComponent implements Rule
      */
     private $target;
 
+    /**
```

Comment:
- [ ] Issues are being reported here https://github.com/oat-sa/qti-sdk/pull/439/files, please move the fixes here when applied in the other branch. Thanks!